app/models/export/contracts/extract.rb

```diff
@@ -0,0 +1,13 @@
+module Export
+  class Contracts
+    module Extract
```

Comment:
looks like there are no spec files for this, or `Export::Contracts::Row`. I know there's a fair amount of overlap with the `Invoice` equivalents, but do you think it might be helpful to have some in place for regression protection and also to give a developer confidence when making changes to these in the future? appreciate they may end up very similar to the `Invoice` ones, but I don't think that's necessarily a bad thing.

Reply:
I will add one.

Reply:
I have added an `extract_spec`. Reviewing `invoices/row_spec.rb` it seems that that file is stuffed with design-time specs that helped in mapping when `value_for` didn't exist, but that those tests are not adding much value now. Probably a better thing would be to delete `invoices/row_spec.rb` in favour of adding a small spec for `SubmissionEntryRow`. What do you think?

Reply:
sounds good to me.

Reply:
Done in https://github.com/dxw/DataSubmissionServiceAPI/pull/73/commits/6821b9a6e73e6d4cc98c466cc4f058e4bd10b772